### PR TITLE
Define invalid attempt

### PR DESCRIPTION
**Pull request by CarterKoala**

### CODE REVIEW

File: wca-regulations.md [hunk 1]
```diff
@@ -272,6 +272,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 9j) Each event must be held at most once per competition.
 - 9k) All competitors may participate in all events of a competition, except in cases specifically approved by the WCA Competition Announcement Team.
 - 9l) Each round must be completed before any following round of the same event can start. Exception: Extra attempts may be given at any point during a competition to replace an attempt that was initially invalid.
+    9l1) An attempt is considered invalid if an extra attempt is required to be granted.
 - 9m) Events must have at most four rounds.
     - 9m1) Rounds with 99 or fewer competitors must have at most two subsequent rounds.
```
Comment (CarterKoala) [inline wca-regulations.md]:
```suggestion
    - 9l1) An attempt is considered invalid if an extra attempt is required to be granted.
- 9m) Events must have at most four rounds.
    - 9m1) Rounds with 99 or fewer competitors must have at most two subsequent rounds.
```

**Comment by EdHollingdale** [comment]
I think this is unnecessarily harsh. I think we should consider an attempt invalid at the discretion of a delegate if an extra would otherwise have been provided per A7g (including for invalid puzzles).

File: wca-regulations.md [hunk 2]
```diff
@@ -272,6 +272,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - 9j) Each event must be held at most once per competition.
 - 9k) All competitors may participate in all events of a competition, except in cases specifically approved by the WCA Competition Announcement Team.
 - 9l) Each round must be completed before any following round of the same event can start. Exception: Extra attempts may be given at any point during a competition to replace an attempt that was initially invalid.
```
Comment (CarterKoala) [inline wca-regulations.md:273]:
```suggestion
- 9l) Each round must be completed before any following round of the same event can start. Exception: Extra attempts may be given at any point during a competition under [A7g](regulations:regulation:A7g) or to replace an attempt that was initially invalid.
```

File: wca-guidelines.md [hunk 3]
```diff
@@ -129,6 +129,8 @@ To be more informative, each Guideline is classified using one of the following
 - 9f2+) [ADDITION] For 3x3x3 Multi-Blind, the judge should record the number of puzzles solved, the number of puzzles attempted, and the final time. For example, if the competitor solves 3 cubes out of 5 attempted cubes in 40 minutes and 30 seconds, the judge should write down 3/5 40:30.
 - 9f4+) [CLARIFICATION] The result of an attempt should be a DNF if the competitor began the attempt by indicating that they are ready (see [Regulation A3b2](regulations:regulation:A3b2)) but the attempt was disqualified.
 - 9f5+) [CLARIFICATION] The result of an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. in a Cutoff Round), they have no result for the attempt.
+- 9l1+) [CLARIFICATION] A DNF Result is considered a valid result. Extra attempts must not be given for using a non-permitted puzzle that is only found after the round is complete (see [Regulation 3k2a](regulations:regulation:3k2a)).
```
Comment (CarterKoala) [inline wca-guidelines.md]:
```suggestion
- 9l1+) [CLARIFICATION] A DNF Result is generally considered a valid result. Extra attempts must not be given for using a non-permitted puzzle that is only found after the round is complete (see [Regulation 3k2a](regulations:regulation:3k2a)).
```
It was pointed out there are some cases where DNFs are not valid results (e.g. misscramble).
If we are adding the A7g thing to the reg, this guideline may need rewording

File: wca-regulations.md (hunk 2 above)
Comment (ohexter) [reply wca-regulations.md:273]: As discussed, not progressing with this for now

File: wca-guidelines.md [hunk 4]
```diff
@@ -129,6 +129,8 @@ To be more informative, each Guideline is classified using one of the following
...
 - 9f4+) [CLARIFICATION] The result of an attempt should be a DNF if the competitor began the attempt by indicating that they are ready (see [Regulation A3b2](regulations:regulation:A3b2)) but the attempt was disqualified.
 - 9f5+) [CLARIFICATION] The result of an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. in a Cutoff Round), they have no result for the attempt.
+- 9l1+) [CLARIFICATION] A DNF Result is generally considered a valid result. Extra attempts must not be given for using a non-permitted puzzle that is only found after the round is complete (see [Regulation 3k2a](regulations:regulation:3k2a)).
+- 9l1++) [EXAMPLE] Examples of an invalid attempt: an attempt that was misscrambled, attempts where the results are no longer retrievable (e.g. a missing scorecard).
```
Comment (ohexter) [inline wca-guidelines.md]:
```suggestion
- 9l1++) [EXAMPLE] Examples of an invalid attempt: an attempt that was incorrectly scrambled, attempts where the results are no longer retrievable (e.g. a missing scorecard).
```
Align wording with misscrambles proposal.

File: wca-guidelines.md [hunk 5]
```diff
@@ -129,6 +129,8 @@ To be more informative, each Guideline is classified using one of the following
 - 9f2+) [ADDITION] For 3x3x3 Multi-Blind, the judge should record the number of puzzles solved, the number of puzzles attempted, and the final time. For example, if the competitor solves 3 cubes out of 5 attempted cubes in 40 minutes and 30 seconds, the judge should write down 3/5 40:30.
 - 9f4+) [CLARIFICATION] The result of an attempt should be a DNF if the competitor began the attempt by indicating that they are ready (see [Regulation A3b2](regulations:regulation:A3b2)) but the attempt was disqualified.
 - 9f5+) [CLARIFICATION] The result of an attempt is DNS if the competitor was eligible for the attempt and did not start it (see [Regulation A3b2](regulations:regulation:A3b2)). If the competitor did not qualify for an attempt (e.g. in a Cutoff Round), they have no result for the attempt.
+- 9l1+) [CLARIFICATION] A DNF Result is generally considered a valid result. Extra attempts must not be given for using a non-permitted puzzle that is only found after the round is complete (see [Regulation 3k2a](regulations:regulation:3k2a)).
```
Comment (ohexter) [inline wca-guidelines.md:132]: Do we need an A7g exeption here? (noting that 9l still prevents the extras if the next round has started.

**Review by ohexter — APPROVED** [review]

**Pull request merged**